packages/examples/cvat/recording-oracle/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy
```

Comment:
We are using `Poetry` to maintain packages so there is no need in creating a `requirements.txt` file.